eclass/kernel-2.eclass

```diff
@@ -907,7 +907,7 @@ compile_headers_tweak_config() {
 # Fix permissions in tarball
 
 install_universal() {
-	chown -R 0:0 "${WORKDIR}"/* >& /dev/null || die
+	chown -R 0:0 "${WORKDIR}"/* &>/dev/null || die
 	chmod -R a+r-w+X,u+w "${WORKDIR}"/* || die
```

Comment:
ooh, there's a classic typo! would probably preserve the space after `&>` for readability. </nitpick>

Reply:
Nearly all other instances of redirection in the file have no space, so I'd prefer to omit it for consistency rather than rewrite all the other lines, unless that is the coding standard.